### DISCUSSION TITLE
Welcome to biogeochem_testbed_1.1 Discussions!

### DISCUSSION THREAD

**Discussion by wwieder** [wwieder/biogeochem_testbed_1.1#44] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by negin513** [comment]
This is a nice feature.

**Reply by wwieder** [reply]
looks like you can link to issue #42
